c/include/ml-api-common.h

```diff
@@ -405,6 +412,56 @@ const char * ml_error (void);
  */
 const char * ml_strerror (int errnum);
 
+/*************
+ * ML OPTION *
+ *************/
+
+/**
+ * @brief A handle of a ml-option instance.
+ * @since_tizen 7.0
+ */
+typedef void *ml_option_h;
+
+/**
+ * @brief Creates ml-option instance.
+ * @since_tizen 7.0
+ * @remarks The @a option should be released using ml_option_destroy().
+ * @param[out] option Newly created option handle is returned.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ */
+int ml_option_create (ml_option_h *option);
+
+/**
+ * @brief Destroys the ml-option instance.
+ * @details Note that, user should free the allocated value of ml-option in the case that destroy function is not given.
+ * @since_tizen 7.0
+ * @param[in] option The option handle to be destroyed.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ */
+int ml_option_destroy (ml_option_h option);
+
+/**
+ * @brief Sets a new key-value in ml-option instance.
+ * @details key is a const string and value is a void pointer. Note that the value should be valid during single task and be freed after using unless proper destroy function is not given. When duplicated key is given, the corresponding value is updated with the new one.
```

Comment:
the first sentence repeats the info from signature, IMO it's not necessary, but can stay as it is correct,
"unless ... not given" looks like improper double negative, please remove the word "not" - if giving the destroy function removes the need to free the value manually,
you can add `@a` in before the parameter names, so they will be styled differently and stand out from description

Reply:
Great thanks, I fix the grammatical error. And I added some `@a`s as you suggested. PTAL